crates/rethnet_evm_napi/src/blockchain.rs

```diff
@@ -9,6 +9,7 @@ use napi::{
 };
 use napi_derive::napi;
 
+use rethnet::config::DEFAULT_CACHE_DIR;
```

Comment:
I ran into a similar dependency requirement myself. I'd propose that we move defaults to a separate crate, as other crates depending on a top-level crate like `rethnet` for this will quickly result in circular dependencies.

Reply:
Good idea, added `rethnet_defaults` crate in [585a8c3](https://github.com/NomicFoundation/hardhat/pull/4280/commits/585a8c3b3e0add1dd2defd1310d7480b97104ece)